cs-dropbox/src/main/java/io/cloudslang/content/dropbox/actions/folders/CreateFolder.java

```diff
@@ -145,7 +141,6 @@
             })
     public Map<String, String> execute(@Param(value = ENDPOINT, required = true) String endpoint,
                                        @Param(value = ACCESS_TOKEN, required = true, encrypted = true) String accessToken,
-                                       @Param(value = API_VERSION) String apiVersion,
```

Comment:
@Bonczidai It has already a fallback mechanism: if user don't provide version then will go on latest/actual one v2, if specified deprecated one then it goes on deprecated api... all that as a result of a previous review....